Sources/Filters.swift

```diff
@@ -89,6 +89,50 @@ struct StringFilters {
     }
   }
 
+  /// Converts camelCase to snake_case. Takes an optional Bool argument for making the string lower case,
+  /// which defaults to true
+  /// - parameter value: the value to be processed
+  /// - parameter arguments: the arguments to the function, expecting zero or one argument
+  /// - returns: the snake case string
+  /// - throws: FilterError.invalidInputType if the value parameter isn't a string
+  static func camelToSnakeCase(_ value: Any?, arguments: [Any?]) throws -> Any? {
+    let toLower = try parseBool(from: arguments, index: 0, required: false) ?? true
+    guard let string = value as? String else { throw FilterError.invalidInputType }
+
+    let snakeCase = try snakecase(string)
+    if toLower {
+      return snakeCase.lowercased()
```

Comment:
Currently `foo|camelToSnakeCase|lowerCase` and `foo|camelToSnakeCase` are equivalent which I expect that 99% of people will want. You would only need to add a parameter if you wanted snake case with first upper characters, which IMHO will probably be pretty rare. That's why I'd rather keep it as is.